tests/pytests/test_dertype.py

```diff
@@ -132,9 +139,10 @@ def test_negotiate_missing_error(inp):
     (('gradient', 'Eg_man', 2)),
     (('energy', 'Eg_man', 2)),
     (('energy', 'Eg_man', 1)),
+    #(("properties", "hf", 2)),
```

Comment:
Why is this commented?

Reply:
I couldn't decide whether it or I was right when it didn't give me the error I expected. So it's commented as a reminder that that error pathway isn't tested. Not a great situation, but line is pretty harmless.

Reply:
If we can get a comment briefly explaining this? I know, you're on the odds-on favorite to probe deeper here, but just in case...

Reply:
sure, done locally.